frontend/src/components/admin-page/components/admin-table/helpers/getRows.helper.ts

```diff
@@ -0,0 +1,71 @@
+import ActionsButton from 'components/admin-page/components/actions-button/actions-button'
```

Comment:
use `cebab-case` for file/folder name

each helper should be in the separate file